draft-ietf-rtcweb-jsep.xml

```diff
@@ -556,6 +556,82 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
       </section>
 
+      <section anchor="sec.imageattr" title="Video Size Negotiation">
+        <t>Video size negotiation is the process through which a receiver can
+          use the "a=imageattr" SDP attribute <xref target="RFC6236"/> to indicate
+          what video frame sizes it is capable of receiving. A receiver may have
+          hard limits on what its video decoder can process, or it may wish to
+          constrain what it receives due to application preferences, e.g. a
+          specific size for the window in which the video will be displayed.
+        </t>
+
+        <section title="Creating an imageattr Attribute">
+          <t>In order to determine the limits on what video resolution a
+            receiver wants to receive, it will intersect its decoder hard
+            limits with any mandatory constraints that have been applied to
+            the associated MediaStreamTrack. For the answerer, these mandatory
+            constraints can be applied to the remote MediaStreamTracks that
+            are created by a setRemoteDescription call, and will affect the
+            output of the ensuing createAnswer call. For the offerer, since it
+            does not know about any remote MediaStreamTracks until it receives
+            theanswer, this implies that it can only set these constraints after
+            the initial offer-answer exchange is complete, and as a result, a
+            new offer-answer is needed to communicate them.</t>
```

Comment:
I think you may be interpreting the text here different from what is intended.

Say that we take the example you give except that once A calls setRD(answer) on B's answer, the JS applies a constraint to the resulting media stream that restricts it to 360p. There is no way for A to communicate that to B in this exchange, because it has concluded, and B is left thinking that it can send at any rate. In order to adjust B's ideas, A needs to send a new offer with a new a=imageattr. Do you disagree with this?